build.rs

```diff
@@ -0,0 +1,33 @@
+use std::io::Result;
+use std::path::Path;
+
+fn main() -> Result<()> {
```

Comment:
I think it's fine to have this here, instead of moving `InternalSerialization.proto` in `libsignal-service-rs`